modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java

```diff
@@ -4176,7 +4176,7 @@ private void _addOrUpdateTaxonomyVocabularies(
 				resourcePath, _servletContext);
 
 			TaxonomyVocabulary taxonomyVocabulary = TaxonomyVocabulary.toDTO(
-				json);
+				_replace(json, serviceContext));
```

Comment:
@nNilton this is the only change in the BundleSiteInitializer to replace the group key.